docs/dev/projectsetup/java_api_quickstart.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Project Template for Java"
 nav-title: Project Template for Java
-nav-parent_id: start
+nav-parent_id: projectsetup
 nav-pos: 0
 ---
 <!--
```

Comment:
Please note that link to `run_example_quickstart.html#writing-a-flink-program` is broken because you move it.
There might be other such broken links, please check :-)

Reply:
Right! Thanks :-)